pkg/ui/ccl/src/views/clusterviz/containers/map/nodeView.tsx

```diff
@@ -45,23 +46,51 @@ export class NodeView extends React.Component<NodeViewProps> {
     return liveIcon;
   }
 
-  render() {
-    const { node, liveness } = this.props;
-    const { capacityUsable, capacityUsed, nodeCounts } = sumNodeStats([node], liveness);
+  getUptimeText() {
+    const { node, livenessStatus, liveness } = this.props;
+
+    const thisLiveness = livenessStatus[node.desc.node_id];
 
-    const startTime = LongToMoment(node.started_at);
-    const uptimeText = "up for " + moment.duration(startTime.diff(moment())).humanize();
+    switch (thisLiveness) {
+      case NodeLivenessStatus.DEAD: {
+        if (!liveness) {
+          return "no information";
+        }
+
+        const deadTime = liveness.expiration.wall_time;
+        const deadMoment = LongToMoment(deadTime);
+        return `dead for ${moment.duration(deadMoment.diff(moment())).humanize()}`;
+      }
+      case NodeLivenessStatus.LIVE: {
+        const startTime = LongToMoment(node.started_at);
+        return "up for " + moment.duration(startTime.diff(moment())).humanize();
+      }
+      case NodeLivenessStatus.DECOMMISSIONED:
+        return "decommissioned";
+      case NodeLivenessStatus.DECOMMISSIONING:
+        return "decommissioning";
+      case NodeLivenessStatus.UNKNOWN:
+        return "unknown";
+      case NodeLivenessStatus.UNAVAILABLE:
+        return "unavailable";
+      default:
+        return ""; // idk man
```

Comment:
probably better to call this "unknown"???

Reply:
Also addressed by using `livenessNomenclature` function.